engines/python/src/main/java/ai/djl/python/engine/PyModel.java

```diff
@@ -150,6 +151,21 @@ public void load(Path modelPath, String prefix, Map<String, ?> options) throws I
             pyEnv.setFailOnInitialize(false);
         }
 
+        // Handle TRT-LLM
+        if ("TRT-LLM".equals(Utils.getenv("LMI_BACKEND")) || Boolean.parseBoolean(getProperty("option.trt_llm"))) {
```

Comment:
env var is always set in the container and we don't want to introduce other option. I will update this,